mitzu/adapters/databricks_adapter.py

```
@@ -43,6 +43,7 @@ def map_type(self, sa_type: Any) -> M.DataType:
     def _parse_map_type(
         self, sa_type: Any, name: str, event_data_table: M.EventDataTable
     ) -> M.Field:
+        assert event_data_table.discovery_settings is not None
```

please avoid production code asserts ... rather throw an exception. 
https://towardsdatascience.com/2-caveats-to-avoid-when-using-python-assert-statement-8060e123c26a

---

https://github.com/mitzu-io/mitzu/pull/66/commits/b10141a2efd1923e4faa4333158ab57e2baa8170